src/scss/global.scss

```diff
@@ -48,29 +48,37 @@ h6 {
   font-weight: 700;
   margin-bottom: 0.5rem;
   line-height: 1.2;
+
   &.primary {
     color: $primary;
   }
+
   &.italic {
     font-weight: 900;
     font-style: italic;
   }
 }
+
 h1 {
   font-size: 3em;
 }
+
 h2 {
   font-size: 2em;
 }
+
 h3 {
   font-size: 1.6em;
 }
+
 h4 {
   font-size: 1.4em;
 }
+
 h5 {
   font-size: 1.2em;
 }
+
 h6 {
   font-size: 1em;
 }
```

Comment:
[OOS] All of these should probably be replaced with something like
`<div className="mod--font-10em">`
`<div className="mod--font-12em">`
...
`<div className="mod--font-20em">`